### PR TITLE
bump dependencies version

### PR DESCRIPTION
**Pull request by Et7f3**
bump dependencies version
fix 2 warnings
fix a typo

### CODE REVIEW

File: src/Angle.re [hunk 1]
```diff
@@ -22,9 +22,9 @@ let to_degrees = (angle: t) => {
 };
 
 let cos = (angle: t) => {
-    Pervasives.cos(to_radians(angle));
+    cos(to_radians(angle));
```
Comment (Et7f3) [inline src/Angle.re:25]: Pervasives is not needed because we haven't rec flag
And it will force to add stdlib-shim

File: package.json [hunk 2]
```diff
@@ -7,16 +7,16 @@
     "build:js": "esy b dune build ./test/bin/test_runner.bc.js",
     "test:js": "node bin/test-jsoo.js",
     "test:native": "esy b dune runtest",
-    "test": "npm run test:js && npm run test:native"
+    "test": "npm run build:js && npm run test:js && npm run test:native"
```
Comment (Et7f3) [inline package.json:10]: if it is build dune will do nothing so it is a zero cost addition

File: package.json [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
   "esy": {
     "build": ["refmterr dune build -p reglm"],
     "buildsInSource": "_build"
   },
   "resolutions": {
     "@esy-ocaml/esy-installer": "0.0.1",
-    "@opam/cmdliner": "1.0.2",
-    "@opam/dune": "1.7.3"
+    "@opam/cmdliner": "1.0.4",
+    "@opam/camomile": "1.0.1"
```
Comment (Et7f3) [inline package.json:19]: cmdliner 1.0.2 or 1.0.4 compile on windows but 1.0.3 don't same for camomile 1.0.2

File: src/dune [hunk 4]
```diff
@@ -3,4 +3,4 @@
     (public_name reglm)
     (js_of_ocaml (javascript_files gl-matrix_stubs.js quat.js))
     (cxx_names glm_stubs)
-    (cxx_flags ((-I ./../../../include/glm) (-fno-exceptions) (-fno-rtti) (-lstdc++) (-x c++))))
+    (cxx_flags ((-I ./../../../include/glm) (-fno-exceptions) (-fPIC) (-fno-rtti) (-lstdc++) (-x c++))))
```
Comment (Et7f3) [inline src/dune:6]: see https://bryphe.visualstudio.com/reason-gl-matrix/_build/results?buildId=1695&view=logs&jobId=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&taskId=6c939d89-0d1a-51f2-8b30-091a7a82e98c&lineStart=117&lineEnd=117&colStart=130&colEnd=135

**Comment by bryphe** [comment]
Thanks for the fix, @Et7f3 ! Looks great 🎉 I'll publish a new version with this.

File: src/Angle.re (hunk 1 above)
Comment (bryphe) [reply src/Angle.re:25]: Cool, thank you for the explanations! 👍

File: src/dune (hunk 4 above)
Comment (bryphe) [reply src/dune:6]: Nice catch!

**Pull request merged**